### ISSUE TITLE
Question on the canceled activation function

### ISSUE THREAD

**Issue by VivianLiang1108** [liuquande/FedIRM#1]
Hi Quande,

Thanks for your contribution! May I ask why did you cancel the activation function (sigmoid here) in Densenet 121?
https://github.com/liuquande/FedIRM/blob/1c86c9cb3ba6a7cef285c88dda6719123916ad96/networks/models.py#L29

Many Thanks!

Best,
Vivian

**Comment by HongzhengYang** [comment]
Hi, Liang
We implemented the DensetNet model to directly output logits. Then the logits were further used to compute CE loss in local_supervised.py and relation matrix in local_unsupervised.py.
Best
